modules/local/report.nf

```diff
@@ -364,15 +366,24 @@ def create_action_call(sample_data){
                 qual_message.add("[PASSED] All Checks passed")
                 sample_status = "PASSED"
             }
-            qual_message.add("Passed Tests: ${checks - checks_failed - checks_ignored}/${checks}")
 
+            def organism_criteria = sample_data[val.key][species_tag]
+            def tests_passed = "Passed Tests: ${checks - checks_failed - checks_ignored}/${checks}"
+            qual_message.add(tests_passed)
 
-            qual_message.add("Species ID: ${val.value[val.key][params.top_hit_species.report_tag]}")
+            def species_id = "Species ID: ${val.value[val.key][params.top_hit_species.report_tag]}"
+            qual_message.add(species_id)
 
             // Qual summary not final message
             final_message = qual_message.join("\n")
             def terminal_message = populate_qual_message(qual_data).join("\n")
             log.info "\n$val.key\n${terminal_message}\n${sample_status}\n${final_message}"
+
+            // Reseq recommended should go to a seperate field
+            // Requested output should be: [PASS|FAILED] Species ID: [species] [Tests passed] [Organism criteria available]
+            qc_message = "${sample_status} ${species_id} ${tests_passed} Organism QC Criteria: ${organism_criteria}"
```

Comment:
Could you maybe add separators between different messages? For example, maybe a semicolon `;`?

Reply:
Fixed in: [62e879d](https://github.com/phac-nml/mikrokondo/pull/85/commits/62e879d37197b56cabdcb1218e29a280a2288bb9)